homeassistant/components/notion/__init__.py

```diff
@@ -137,13 +138,12 @@ def __init__(
         sensor_id: str,
         bridge_id: str,
         system_id: str,
-        name: str,
-        device_class: str,
+        description: EntityDescription,
     ) -> None:
         """Initialize the entity."""
         super().__init__(coordinator)
 
-        self._attr_device_class = device_class
+        self._attr_device_class = description.device_class
```

Comment:
Assign `self.entity_description = description` instead. With that, you don't need this anymore.